native/packages/react-native-bpk-component-banner-alert/src/BpkBannerAlert.android.js

```diff
@@ -0,0 +1,255 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  StyleSheet,
+  TouchableNativeFeedback,
+  TouchableWithoutFeedback,
+  View,
+  ViewPropTypes,
+} from 'react-native';
+
+import {
+  borderRadiusSm,
+  colorBlue500,
+  colorGray50,
+  colorGray500,
+  colorGreen500,
+  colorRed500,
+  colorYellow500,
+  spacingBase,
+  spacingLg,
+  spacingMd,
+  spacingSm,
+  spacingXxl,
+} from 'bpk-tokens/tokens/base.react.native';
+
+import BpkAnimateHeight from 'react-native-bpk-component-animate-height';
+import BpkIcon from 'react-native-bpk-component-icon';
+import BpkText from 'react-native-bpk-component-text';
+
+import AnimateAndFade from './AnimateAndFade';
+import {
+  dismissablePropType,
+  toggleExpandedButtonLabelPropType,
+  dismissableLabelPropType,
+} from './customPropTypes';
+
+export const ALERT_TYPES = {
+  SUCCESS: 'success',
+  WARN: 'warn',
+  ERROR: 'error',
+  NEUTRAL: 'neutral',
+};
+
+const styles = StyleSheet.create({
+  mainContainer: {
+    paddingVertical: spacingBase,
+    paddingHorizontal: spacingLg,
+    backgroundColor: colorGray50,
+    minHeight: spacingXxl + spacingBase,
+    flex: 1,
+  },
+  dismissableContainer: {
+    paddingHorizontal: spacingLg - spacingMd,
+    backgroundColor: colorGray50,
+    justifyContent: 'center',
+  },
+  childrenContainer: {
+    paddingVertical: spacingSm,
+  },
+  flexRow: {
+    display: 'flex',
+    flexDirection: 'row',
+  },
+  text: {
+    flex: 1,
+  },
+  icon: {
+    paddingRight: spacingSm,
+    paddingTop: spacingSm - spacingSm / spacingSm,
```

Comment:
Huh? This is just `spacingSm - 1` right?

Reply:
yep, reason being, the icon renders with some spacing up top, not sure why but this does it 😄

Reply:
I'd be tempted to just make that `spacingSm - 1` in that case, it feels a bit like we're going out of our way to use a token when `1` would make more sense (if it were a number other than 1 I'd say use a token though)

Reply:
But why not write it as `spacingSm - 1` though? All though we should investigate why the spacing is there and get rid of it

Reply:
I thought magic arbitrary numbers were banned, I'll amend to `1`